.github/workflows/test.yml

```diff
@@ -19,22 +19,46 @@ jobs:
     - name: Setup Gradle
       uses: gradle/gradle-build-action@v2
     
-    - name: Run Tests with Coverage
+    - name: Run Tests with Coverage (jpo-asn-j2735-2024)

```

Comment:
question: with the latest update to include both j2735 and runtime tests in the same run, should this reference both for clarity?

Reply:
The github workflow uses gradle to build and post coverage comments to the PRs.  Maven jacoco is configured to aggregate, but the gradle version of jacoco doesn't aggregate: it only shows the coverage for one project.  Since this PR adds the test-generator project, it adds another task to display coverage for it, but that label is accurate as is, since the runtime project coverage isn't shown in those comments.

Reply:
It turns that that Gradle uses a separate plugin to do coverage aggregation.  I added that in and changed the workflow and readme instructions to match.  (You need to run 'gradle check' instead of 'gradle test' for it to work).  I tested that it works in a separate PR here: https://github.com/CDOT-CV/jpo-asn-pojos/pull/25